target/linux/pistachio/modules.mk

```diff
@@ -1,3 +1,27 @@
+define KernelPackage/sound-pistachio-soc
```

Comment:
@francois-berder-imgtec this is the OpenWRT technique to add the audio support via a kernel module - the built in support for the audio in target/linux/pistachio/config-4.1 is incorrect. @mayank-sirotiya-imgtec this is the approach you need to use for #20 - building into the kernel is not the OpenWRT way.